compilation_flags_lto.file

```diff
@@ -1,11 +1,19 @@
 #Enable/disable LTO builds
 %define enable_lto 1
 
-#LTO related flags
+#Disable LTO for ppc64le
 %ifarch ppc64le
 %undefine enable_lto
 %define enable_lto 0
 %endif
+
+#Disable LTO for ASAN
+%if "%(case %realversion in (*_ASAN_X*) echo true ;; (*) echo false ;; esac)" == "true"
+%undefine enable_lto
+%define enable_lto 0
+%endif
+
```

Comment:
@aandvalenzuela , this is not going to work. this file is not directly used by cmssw so `realversion` will not container `ASAN`. I would suggest to just change `%define enable_lto 1` to `%define enable_lto 0` on the 2nd line of this file and try to build full cmssw for ASAN

Reply:
Ah, nice! In fact, I was a bit unsure of how to filter based on flavor in this file. Do I make the change, trigger the IB, and revert the change?

Reply:
No, no need to build the IB. Just the PR tests should be enough to see if LTO is causing any issues for ASAN